packages/core/src/endowmentsToolkit.js

```diff
@@ -20,9 +20,14 @@ module.exports = endowmentsToolkit
 /**
  * @param {object} opts
  * @param {DefaultWrapperFn} [opts.createFunctionWrapper]
+ * @param {boolean} [opts.handleGlobalWrite]
+ * @param {Set<string>} [opts.knownWritableFields] - List of globals that can be
+ *   mutated later
  */
 function endowmentsToolkit({
   createFunctionWrapper = defaultCreateFunctionWrapper,
+  handleGlobalWrite = false,
+  knownWritableFields = new Set(),
```

Comment:
global write is a decision to make once per implementation, so it belongs here along with the configuration.